test/e2e/flavors/curatedpackages/utils.go

```diff
@@ -0,0 +1,200 @@
+//go:build e2e
+// +build e2e
+
+package curatedpackages
```

Comment:
seems like this is a leftover? seem like a duplicate with test/e2e/curatedpackages.go

Reply:
Good catch, removed it.